### PR TITLE
feat: add scaleway security group module

### PR DESCRIPTION
**Pull request by abarbare**
##### SUMMARY

This module implements the scaleway Security Group management module. 
It allows user to:
- create a Security Group
- delete a Security Group

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

scaleway_security_group

##### ANSIBLE VERSION

```paste below
ansible 2.8.0.dev0 (security_groups f11c86630c) last updated 2018/09/16 12:12:24 (GMT +200)
  config file = /Users/abarbare/.ansible.cfg
  configured module search path = [u'/Users/abarbare/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/abarbare/Documents/ansible/lib/ansible
  executable location = /Users/abarbare/Documents/ansible/bin/ansible
  python version = 2.7.15 (default, Jul 23 2018, 21:27:06) [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
```

### CODE REVIEW

**Comment by pilou-** [comment]
peer reviewed with submitter and tested with my own credentials.

LGTM

File: lib/ansible/modules/cloud/scaleway/scaleway_security_group.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        ret['scaleway_security_group'] = security_group_lookup[security_group['name']]
+
+    return ret
+
+
+def absent_strategy(api, security_group):
+    response = api.get('security_groups')
+    ret = {'changed': False}
+
+    security_groups = response.json['security_groups']
```
Comment (mscherer) [inline lib/ansible/modules/cloud/scaleway/scaleway_security_group.py]: So, if the answer is not ok, how can we be sure that response.json will return something ?

Comment (abarbare) [reply lib/ansible/modules/cloud/scaleway/scaleway_security_group.py]: Thanks !

**Comment by pilou-** [comment]
LGTM :)

**Comment by mscherer** [comment]
shipit

**Comment by pilou-** [comment]
@abarbare a rebase is required

**Comment by abarbare** [comment]
Hello @gundalow can you have a look on this one ? 
Thanks

**Comment by abarbare** [comment]
bot_status

**Comment by pilou-** [comment]
shipit

**Comment by mscherer** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Thank you all for the code and reviews. Merged into `devel` for release in Ansible 2.8

**Pull request merged**